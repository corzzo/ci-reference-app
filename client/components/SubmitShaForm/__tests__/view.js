// import React from 'react'
// import SubmitShaForm from '../View'
// import renderer from 'react-test-renderer'

xit('renders correctly', () => {

})
