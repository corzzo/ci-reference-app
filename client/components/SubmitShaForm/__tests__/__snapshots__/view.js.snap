// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  className=""
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    required={true}
    type="text"
    value=""
  />
  <button
    type="submit"
  >
    Submit
  </button>
  <table>
    <thead>
      <tr>
        <th>
          Shas
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          1
        </td>
      </tr>
      <tr>
        <td>
          2
        </td>
      </tr>
    </tbody>
  </table>
</form>
`;
